ert/storage/_storage.py

```diff
@@ -99,28 +110,109 @@ async def _load_blob_record(self) -> ert.data.BlobRecord:
         return ert.data.BlobRecord(data=record.data)
 
 
+async def get_record_storage_transmitters(
```

Comment:
1. Could this get some documentation?
2. Is this how you get a uniform record, i.e. multiple records given an `ensemble_size` and `record_source`? Why isn't this implemented for all transmitters?

Reply:
> Could this get some documentation? 

Sure what would do you feel needs more documentation?

> Is this how you get a uniform record, i.e. multiple records given an ensemble_size and record_source? 

Yes that is the main idea

> Why isn't this implemented for all transmitters? 

Why would it be useful to make file transmitters or in memory transmitters from records that are already in storage ?

Reply:
> Why would it be useful to make file transmitters or in memory transmitters from records that are already in storage ?

Let me rephrase: Is uniform records only supported for ert-storage?

Reply:
In my world view "Uniform records" are a special type of `EnsembleRecords` which currently are only supported for ert-storage so I guess yes they are only supported for ert-storage

Reply:
So if I want to use uniform shared-disk records, they currently work (the records themselves are copied _n_ times), but they aren't implemented the same way as the ert-storage records?

Reply:
There is no implementation of the `RecordCollection` (former `EnsembleRecord`) that accounts for any transmitters currently. My PR just makes the previous code that was storing these RecordCollection to storage use transmitters, also getting these record collections from storage. And I guess because we are storing records in storage we use StorageTrasmitters.

There is no such thing as uniform shared-disk records, I am not sure there should be